client/widgets/org-stats/stats.directive.js

```diff
@@ -0,0 +1,89 @@
+angular.module('probedock.orgStatsWidget').directive('orgStatsWidget', function() {
+  return {
+    restrict: 'E',
+    controller: 'OrgStatsWidgetCtrl',
+    templateUrl: '/templates/widgets/org-stats/stats.template.html',
+    scope: {
+      organization: '='
+    }
+  };
+}).controller('OrgStatsWidgetCtrl', function(api, $scope, orgs) {
+  orgs.addAuthFunctions($scope);
+
+  _.defaults($scope, {
+    params: {
+      organizationId: $scope.organization.id
```

Comment:
This can produce an NPE when no organization is pre-selected (`$scope.organization` can be null).